### ISSUE TITLE
Add missing SU data items

### ISSUE THREAD

**Issue by vaitkus** [COMCIFS/magnetic_dic#63]
All measurand data items are expected to have a corresponding SU data item. Currently, the following measurand items do not:
```
_atom_site_moment.cartn
_atom_site_moment.cartn_x
_atom_site_moment.cartn_y
_atom_site_moment.cartn_z
_atom_site_moment.crystalaxis
_atom_site_moment.crystalaxis_x
_atom_site_moment.crystalaxis_y
_atom_site_moment.crystalaxis_z
_atom_site_moment.magnitude
_atom_site_moment.spherical_azimuthal
_atom_site_moment.spherical_modulus
_atom_site_moment.spherical_polar
_atom_site_rotation.cartn
_atom_site_rotation.cartn_x
_atom_site_rotation.cartn_y
_atom_site_rotation.cartn_z
_atom_site_rotation.crystalaxis
_atom_site_rotation.crystalaxis_x
_atom_site_rotation.crystalaxis_y
_atom_site_rotation.crystalaxis_z
_atom_site_rotation.magnitude
_atom_site_rotation.spherical_azimuthal
_atom_site_rotation.spherical_modulus
_atom_site_rotation.spherical_polar
_atom_site_moment_fourier_param.cos
_atom_site_moment_fourier_param.modulus
_atom_site_moment_fourier_param.phase
_atom_site_moment_fourier_param.sin
_atom_site_moment_special_func.sawtooth_ax
_atom_site_moment_special_func.sawtooth_ay
_atom_site_moment_special_func.sawtooth_az
_atom_site_moment_special_func.sawtooth_c
_atom_site_moment_special_func.sawtooth_w
```
If I recall correctly, @jamesrhester has a piece of software capable of adding these (unfortunately, I could not get it running on my setup). It might also be worthwhile to go through this list and to double check if all of these are indeed measurands (measured values with a standard uncertainty).

**Comment by brantonc** [comment]
I confirm these are all measurands or vectors containing measurands.  The units for the _atom_site_moment_special_func.sawtooth_w/c items are not explicitly stated (radians vs cycles); but then neither are the analogous items in the mscif dictionary.

Branton

From: Antanas Vaitkus ***@***.***>
Sent: Friday, January 19, 2024 7:12 AM
To: COMCIFS/magnetic_dic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [COMCIFS/magnetic_dic] Add missing SU data items (Issue #63)


All measurand data items are expected to have a corresponding SU data item. Currently, the following measurand items do not:

_atom_site_moment.cartn

_atom_site_moment.cartn_x

_atom_site_moment.cartn_y

_atom_site_moment.cartn_z

_atom_site_moment.crystalaxis

_atom_site_moment.crystalaxis_x

_atom_site_moment.crystalaxis_y

_atom_site_moment.crystalaxis_z

_atom_site_moment.magnitude

_atom_site_moment.spherical_azimuthal

_atom_site_moment.spherical_modulus

_atom_site_moment.spherical_polar

_atom_site_rotation.cartn

_atom_site_rotation.cartn_x

_atom_site_rotation.cartn_y

_atom_site_rotation.cartn_z

_atom_site_rotation.crystalaxis

_atom_site_rotation.crystalaxis_x

_atom_site_rotation.crystalaxis_y

_atom_site_rotation.crystalaxis_z

_atom_site_rotation.magnitude

_atom_site_rotation.spherical_azimuthal

_atom_site_rotation.spherical_modulus

_atom_site_rotation.spherical_polar

_atom_site_moment_fourier_param.cos

_atom_site_moment_fourier_param.modulus

_atom_site_moment_fourier_param.phase

_atom_site_moment_fourier_param.sin

_atom_site_moment_special_func.sawtooth_ax

_atom_site_moment_special_func.sawtooth_ay

_atom_site_moment_special_func.sawtooth_az

_atom_site_moment_special_func.sawtooth_c

_atom_site_moment_special_func.sawtooth_w

If I recall correctly, @jamesrhester<https://github.com/jamesrhester> has a piece of software capable of adding these (unfortunately, I could not get it running on my setup). It might also be worthwhile to go through this list and to double check if all of these are indeed measurands (measured values with a standard uncertainty).

**Comment by jamesrhester** [comment]
Thanks for the confirmation @brantonc . I will generate a PR.

**Comment by vaitkus** [comment]
Fixed by merging PR #81.
